packages/manager/modules/iam/src/components/resourceSelect/resourceSelect.template.html

```diff
@@ -17,6 +17,7 @@
             data-multiple="true"
             data-name="{{:: $ctrl.resourceTypesName }}"
             data-on-change="$ctrl.onResourceTypesChanged(modelValue)"
+            data-on-confirm-remove="$ctrl.resourceTypesConfirmRemove({ item, index })"
```

Comment:
it's a good practice to prefix event with 'on' keyword.
```suggestion
            data-on-confirm-remove="$ctrl.onResourceTypesConfirmRemove({ item, index })"
```